### ISSUE TITLE
Teacher Tips may need updates

### ISSUE THREAD

**Issue by terracoda** [phetsims/forces-and-motion-basics#509]
@arouinfar and @Nancy-Salpepi, this is likely a general problem for sims being published with “Core Description.” I just happen to notice that the Teacher Tips document does not have a link to the A11y View for this sim.

Past practice is that when we publish sims with Interactive Description we always provide a link to the the A11y View in the Teacher Tips. I think this should be the same for sims going out with any significant amount of description, like "Core Description". It is meant to allow teachers to see the descriptions and decide if the content in the A11y View meets the needs of their planned lessons.

It has also been past practice to include an Inclusive Features section that describes the features and anything special about them. 

Here's a couple of examples
- https://phet.colorado.edu/en/simulations/ohms-law/teaching-resources
- https://phet.colorado.edu/en/simulations/friction/teaching-resources

Some Teacher Tips get slightly re-worded heading structures due to the nature of their features
- https://phet.colorado.edu/en/simulations/quadrilateral/teaching-resources

**Comment by arouinfar** [comment]
I updated the FAMB Teacher Tips to use the latest template, which now includes an Inclusive Features section. The updated files have been uploaded to SharePoint and the website, closing.
